tools/slicec-cs/src/cs_options.rs

```diff
@@ -18,6 +18,10 @@ pub struct CsOptions {
     #[arg(long = "rpc", value_enum, default_value_t = RpcProvider::IceRpc, ignore_case = true)]
     pub rpc_provider: RpcProvider,
 
+    /// Enable additional compilation information output.
```

Comment:
Sorry to be a little nitpicky, but I also think that this comment should be reworded.
It sounds like a hold-over from when this was `--verbose`.

If this was internal, I wouldn't care, but `clap` uses these doc comments to generate help messages, suggestions, etc.
So they are directly user-facing.